packages/nodes-base/nodes/QuickBase/RecordDescription.ts

```diff
@@ -527,9 +525,7 @@ export const recordFields = [
 			},
 		},
 		default: '',
-		description: `You're updating records in a Quick Base table with data from an external file. In order for a merge like this to work,<br>
-		Quick Base needs a way to match records in the source data with corresponding records in the destination table. You make this possible by<br>
-		choosing the field in the app table that holds unique matching values. This is called a merge field.`,
+		description: `You're updating records in a Quick Base table with data from an external file. In order for a merge like this to work, Quick Base needs a way to match records in the source data with corresponding records in the destination table. You make this possible by choosing the field in the app table that holds unique matching values. This is called a merge field.`,
```

Comment:
```suggestion
		description: `<p>You're updating records in a Quick Base table with data from an external file. In order for a merge like this to work, Quick Base needs a way to match records in the source data with corresponding records in the destination table. </p><p>You make this possible by choosing the field in the app table that holds unique matching values. This is called a merge field.</p>`,
```

Reply:
weird tooltip that starts with `You`. not sure how to rephrase this

Reply:
Multiline fixed.

Agree but we probably should not dwell on phrasing in this PR. If you like I can rig the nodelinter to extract all param descriptions for someone to review.